src/algebra/polynomial/big_operators.lean

```diff
@@ -140,7 +141,21 @@ lemma nat_degree_prod [nontrivial R] (h : ∀ i ∈ s, f i ≠ 0) :
 begin
   apply nat_degree_prod',
   rw prod_ne_zero_iff,
-  intros x hx, simp [h x hx],
+  intros x hx, simp [h x hx]
+end
+
+lemma nat_degree_multiset_prod [nontrivial R] {s : multiset (polynomial R)}
```

Comment:
`s` is explicit to `nat_degree_prod`, so perhaps for consistency should be here too.

Reply:
Thanks for spotting this. Consistency is nice. Fixed.